src/main/target/OMNIBUSF7/target.h

```diff
@@ -15,25 +15,35 @@
 
 #pragma once
 
+//OMNIBUSF7 TARGETS-------------------------
 #define USE_TARGET_CONFIG
-
-#ifdef OMNIBUSF7V2
+#if defined (FPVMODELF7)
+#define TARGET_BOARD_IDENTIFIER "BFF7"
```

Comment:
FMF7?

Reply:
Good spot!

Reply:
Arha yes, god dam it haha ill get there one day ;)